activesupport/lib/active_support/logger_thread_safe_level.rb

```diff
@@ -40,8 +45,6 @@ def log_at(level)
     end
 
     private
-      def local_level_key
-        @local_level_key ||= :"logger_thread_safe_level_#{object_id}"
-      end
+      attr_reader :local_level_key
```

Comment:
Making this private isn't really necessary since the module is nodoc-ed, but I thought I'd do it for consistency with the current implementation.